From cac858425afadba6c0f926c177988b89b85c5397 Mon Sep 17 00:00:00 2001
From: John Rinehart <johnrichardrinehart@gmail.com>
Date: Sat, 2 Nov 2024 14:45:10 -0700
Subject: [PATCH 4/5] fix: just checkout the dts from linux-6.1-stan-rkr1

This change makes the cpp'ed version match that on vendor's branch.
Turns out there were many commits that affected `rk3588s-rock-5c.dts` -
not just the one that I originally pulled in.
---
 .../boot/dts/rockchip/rk3588s-rock-5c.dts     | 20 ++++++++++++++-----
 1 file changed, 15 insertions(+), 5 deletions(-)

diff --git a/arch/arm64/boot/dts/rockchip/rk3588s-rock-5c.dts b/arch/arm64/boot/dts/rockchip/rk3588s-rock-5c.dts
index e785f7dc9daf..3628870795ea 100644
--- a/arch/arm64/boot/dts/rockchip/rk3588s-rock-5c.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3588s-rock-5c.dts
@@ -154,9 +154,8 @@ user-led1 {
 		};
 
 		user-led2 {
-			gpios = <&gpio3 RK_PD5 GPIO_ACTIVE_LOW>;
+			gpios = <&gpio3 RK_PD5 GPIO_ACTIVE_HIGH>;
 			linux,default-trigger = "heartbeat";
-			default-state = "on";
 		};
 	};
 };
@@ -387,6 +386,14 @@ &rkvenc1_mmu {
 	status = "okay";
 };
 
+&av1d {
+	status = "okay";
+};
+
+&av1d_mmu {
+	status = "okay";
+};
+
 &saradc {
 	status = "okay";
 	vref-supply = <&avcc_1v8_s0>;
@@ -436,6 +443,7 @@ &tsadc {
 &hdmi0 {
 	status = "okay";
 	cec-enable = "true";
+	enable-gpios = <&gpio4 RK_PB6 GPIO_ACTIVE_HIGH>;
 	pinctrl-names = "default";
 	pinctrl-0 = <&hdmim0_tx0_cec &hdmim1_tx0_hpd &hdmim0_tx0_scl &hdmim0_tx0_sda>;
 };
@@ -638,8 +646,10 @@ &i2c7 {
 	es8316: es8316@11 {
 		compatible = "everest,es8316";
 		reg = <0x11>;
-		clocks = <&cru I2S0_8CH_MCLKOUT>;
+		clocks = <&mclkout_i2s0>;
 		clock-names = "mclk";
+		assigned-clocks = <&mclkout_i2s0>;
+		assigned-clock-rates = <12288000>;
 		pinctrl-names = "default";
 		pinctrl-0 = <&i2s0_mclk>;
 		#sound-dai-cells = <0>;
@@ -670,13 +680,13 @@ &threshold {
 &soc_thermal {
 	sustainable-power = <5000>; /* milliwatts */
 	cooling-maps {
-		map3 {
+		map4 {
 			trip = <&target>;
 			cooling-device =
 				<&fan0 THERMAL_NO_LIMIT THERMAL_NO_LIMIT>;
 			contribution = <8192>;
 		};
-		map4 {
+		map5 {
 			trip = <&threshold>;
 			cooling-device =
 				<&fan0 THERMAL_NO_LIMIT THERMAL_NO_LIMIT>;
-- 
2.45.2

